src/liveEditor/globals/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Signal, signal } from "@preact/signals";
+
+interface VisualEditorGlobalStateImpl {
+    // customCursor: HTMLDivElement | null;
+    // overlayWrapper: HTMLDivElement | null;
+    previousSelectedEditableDOM: HTMLElement | Element | null;
+    // visualEditorContainer: HTMLDivElement | null;
+    previousHoveredTargetDOM: Element | null;
+    // focusedToolbar: HTMLDivElement | null;
+}
+
+// const VisualEditorGlobalState.value: VisualEditorGlobalState.valueImpl = {
+//     // customCursor: null,
+//     // overlayWrapper: null,
+//     previousSelectedEditableDOM: null,
+//     // visualEditorContainer: null,
+//     previousHoveredTargetDOM: null,
+//     // focusedToolbar: null
+// };
+
```

Comment:
remove dead code

Reply:
Sure. Would we need to add those commented fields in the global state in the future ? The event handlers still need these to be passed into them.